### DISCUSSION TITLE
How to pass parameter in query?

### DISCUSSION THREAD

**Discussion by Artein** [ecoAPM/LoadTestToolbox#86] [Q&A] [answered]
Hi,

My name is Artem and I have following endpoint `api/front-app-config`. This endpoint has single string parameter.
Here is my C# .Net Core Web API implementation of it:
```
[HttpGet]
public ActionResult<string> GetFrontAppConfig([FromQuery] string environment)
```

Unfortunately, if I type following command I receive error.
Command: `ltt drill -u http://localhost:5206/api/front-app-config?environment=development -r 1 -d 1 -f mocean-front-app-config.png`
Error: `zsh: no matches found: http://localhost:5206/api/front-app-config?environment=development`

But, if I omit parameter everything works great.

What am I doing wrong?

**Comment by SteveDesmond-ca** [comment, marked as answer]
Have you tried putting the URL in "double quotes"?

**Reply by Artein** [reply]
Thank you. Now it works like a charm
